imports/client/components/PuzzlePage.tsx

```diff
@@ -878,7 +878,9 @@ const PuzzleGuessModal = React.forwardRef(({
   }, []);
 
   const onSubmitGuess = useCallback(() => {
-    const repeatGuess = guesses.find((g) => { return g.guess === guessInput; });
+    const repeatGuess = guesses.find((g) => {
+      return g.guess.replaceAll(/\s/, '') === guessInput.replaceAll(/\s/, '');
```

Comment:
nit: can you hoist `guessInput.replaceAll(/\s/, '')` outside the loop so we compute it just once instead of on every loop iteration?